app/views/layouts/foreman_rh_cloud/application.html.erb

```diff
@@ -3,7 +3,6 @@
 <% end %>
 
 <% content_for(:content) do %>
-  <%= notifications %>
```

Comment:
Actually, I think we can remove the whole file as we moved to full react-router pages,
imo this file doesn't even get called, what do you think @ShimShtein ?